app/controllers/SessionTimeoutController.scala

```diff
@@ -17,20 +17,19 @@
 package controllers
 
 import config.FrontendAppConfig
-import javax.inject.{Inject, Singleton}
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import play.api.{Configuration, Environment, Logging}
-import uk.gov.hmrc.play.bootstrap.config.AuthRedirects
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import utils.Session
 
+import javax.inject.{Inject, Singleton}
 import scala.concurrent.Future
 
 @Singleton
 class SessionTimeoutController @Inject()(val appConfig: FrontendAppConfig,
                                          val config: Configuration,
                                          val env: Environment,
-                                         mcc: MessagesControllerComponents) extends FrontendController(mcc) with AuthRedirects with Logging {
+                                         mcc: MessagesControllerComponents) extends FrontendController(mcc) with Logging {
```

Comment:
Can I just check why `AuthRedirects` has gone from here?  Mixed in elsewhere, I guess?

Reply:
As far as I could see non of the methods or val were used in the controller. They removed this from bootstrap recently.